lib/foreman_remote_execution/engine.rb

```diff
@@ -102,8 +102,8 @@ class Engine < ::Rails::Engine
         ]
 
         # Add a new role called 'Remote Execution User ' if it doesn't exist
-        role 'Remote Execution User', USER_PERMISSIONS
-        role 'Remote Execution Manager', MANAGER_PERMISSIONS
+        role 'Remote Execution User', USER_PERMISSIONS, 'Role with permissions to run remote execution jobs against hosts.'
+        role 'Remote Execution Manager', MANAGER_PERMISSIONS, 'Role with permissions to manage job templates, remote execution features, cancel jobs and view audit logs'
```

Comment:
Nitpick: One has `.` at the end and the other one doesn't. I think it should be unified